app/jobs/reports/verification_errors_report.rb

```diff
@@ -0,0 +1,62 @@
+require 'identity/hostdata'
+require 'csv'
+
+module Reports
+  class VerificationErrorsReport < BaseReport
+    REPORT_NAME = 'verification-errors-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(_date)
+      configs = IdentityConfig.store.verification_errors_report_configs
+      configs.each do |report_hash|
+        name = report_hash['name']
+        emails = report_hash['emails']
+        issuers = report_hash['issuers']
+        report = verification_errors_data_for_issuers(issuers)
+        emails.each do |email|
+          UserMailer.verification_errors_report(
+            email: email,
+            name: name,
+            issuers: issuers,
+            data: report,
+          ).deliver_now_or_later
+        end
+      end
+    end
+
+    private
+
+    def verification_errors_data_for_issuers(issuers)
+      csv = CSV.new('', row_sep: "\r\n")
```

Comment:
can we add a header row?
```suggestion
      csv = CSV.new('', row_sep: "\r\n")
      csv << %w[uuid welcome_view_at error_code]
```